pkg/local/vmis.go

```diff
@@ -104,72 +85,68 @@ func createStorageOption(enableMonitoringStorage string) vmov1.Storage {
 	}
 }
 
-// Constructs the necessary VMI for the given VerrazzanoBinding
-func newVMIs(binding *v1beta1v8o.VerrazzanoBinding, verrazzanoURI string, enableMonitoringStorage string) []*vmov1.VerrazzanoMonitoringInstance {
+// Constructs the necessary VerrazzanoMonitoringInstance for the given VerrazzanoBinding
+func createInstance(binding *v1beta1v8o.VerrazzanoBinding, verrazzanoURI string, enableMonitoringStorage string) *vmov1.VerrazzanoMonitoringInstance {
 	bindingLabels := util.GetLocalBindingLabels(binding)
 
-	if verrazzanoURI == "" {
```

Comment:
Should this return error when verrazzanoURI == ""? Otherwise, this changed the behavior